openmdao/docs/openmdao_book/features/debugging/profiling/timing.ipynb

```diff
@@ -31,7 +31,7 @@
     "\n",
     "The simplest way to use it is via the command line using the `openmdao timing` command. For example:\n",
     "``` bash\n",
-    "    openmdao timing <your_python_script_here>\n",
+    "    mpirun -n <nprocs> openmdao timing <your_python_script_here>\n",
```

Comment:
I think the paragraph above is still somewhat ambiguous about whether this tool is applicable when NOT running under MPI:
```
    "There is another way to view timings for methods called on specific group and component instances in \n",
    "an OpenMDAO model.  Not only does it have lower overhead than the previously discussed instance-based \n",
    "profiler, but it also displays, when running your model under MPI, how that model spends its time \n",
    "across MPI processes.\n",```
```
Is this still useful for non-MPI use cases, with the advantage of lower overhead?

The change to the om_command page implies that this is really intended _only_ for MPI use cases.
If so, I think that moving the qualifying clause to the first sentence might be more consistent with that intent:
```
    "There is another way to view timings for methods called on specific group and component instances in \n",
    "an OpenMDAO model when running under MPI.",
    "Not only does it have lower overhead than the previously discussed instance-based \n",
    "profiler, but it also displays how that model spends its time \n",
    "across MPI processes.\n",```
```

Reply:
I reworded the paragraph to make it clear that it's intended to use only under MPI.